tensorflow/core/common_runtime/constant_folding_test.cc

```diff
@@ -19,9 +19,6 @@ limitations under the License.
 #include <vector>
 
 #include "tensorflow/cc/ops/nn_ops.h"
-#include "tensorflow/core/common_runtime/constant_folding.h"
-
-#include "tensorflow/cc/ops/array_ops_internal.h"
```

Comment:
these look like unrelated changes

Reply:
It must be vscode clang-format. It is occassionally adding junk or making some lines disappear. These are not intentional changes.